reactor-extra/src/test/java/reactor/bool/BoolUtilsTest.java

```diff
@@ -0,0 +1,122 @@
+package reactor.bool;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.testng.Assert.*;
+import static reactor.bool.BoolUtils.*;
+
+import org.testng.annotations.Test;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+/**
+ * @author Greg Turnquist
+ */
+public class BoolUtilsTest {
+
+	private static final Mono<Boolean> TRUE = Mono.just(true);
+	private static final Mono<Boolean> FALSE = Mono.just(false);
+
+	@Test
+	public void testNot() {
+		StepVerifier.create(not(TRUE))
+			.assertNext(bool -> assertThat(bool).isFalse());
```

Comment:
nitpicking or tip? in all these tests I'd tend to use `assertThat(...).as("xxx")`, otherwise any failure would look like "expected [tru]e was [fals]e".
eg.
```
StepVerifier.create(not(TRUE))
			.assertNext(bool -> assertThat(bool).as("not(TRUE)").isFalse())
            .verifyComplete();
```